enhancements/proposals/1665-metadata/README.md

```diff
@@ -0,0 +1,122 @@
+# Support for ScyllaDB Alternator
+
+## Summary
+
+This proposal aims to introduce basic support for handling metadata in managed objects by ScyllaClusters.
+It aims to make minimal changes to satisfy the use cases presented so far.
+
+## Motivation
+
+ScyllaCluster users need to specify labels and annotations for various reasons including external integrations or object discovery / tracking.
+
+### Goals
+
+- Inherit ScyllaCluster labels and annotation into managed objects (aka default labels)
+- Allow specifying distinct labels and annotations for node services
```

Comment:
Wouldn't `distinct` suggest you'd be able to specify it per pod/service/ingress?